<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_data_folder_D"/>
    <addaction name="actionOpen_log_folder_L"/>
    <addaction name="actionQuit_Q"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCheck_port"/>
    <addaction name="actionClearPort"/>
    <addaction name="actionCalibration"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuSetting"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_data_folder_D"/>
   <addaction name="actionCheck_port"/>
   <addaction name="actionCalibration"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionOpen_data_folder_D">
   <property name="icon">
    <iconset resource="../resources/rc.qrc">
     <normaloff>:/images/Open.png</normaloff>:/images/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open data folder(&amp;D)</string>
   </property>
  </action>
  <action name="actionOpen_log_folder_L">
   <property name="icon">
    <iconset resource="../resources/rc.qrc">
     <normaloff>:/images/Open.png</normaloff>:/images/Open.png</iconset>
   </property>
   <property name="text">
    <string>Open log folder(&amp;L)</string>
   </property>
  </action>
  <action name="actionQuit_Q">
   <property name="icon">
    <iconset resource="../resources/rc.qrc">
     <normaloff>:/images/Exit.png</normaloff>:/images/Exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit(&amp;Q)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../resources/rc.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCheck_port">
   <property name="icon">
    <iconset resource="../resources/rc.qrc">
     <normaloff>:/images/mdc.png</normaloff>:/images/mdc.png</iconset>
   </property>
   <property name="text">
    <string>Check port</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../resources/rc.qrc">
     <normaloff>:/images/run.png</normaloff>:/images/run.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../resources/rc.qrc">
     <normaloff>:/images/pause_press.png</normaloff>:/images/pause_press.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionCalibration">
   <property name="icon">
    <iconset resource="../resources/rc.qrc">
     <normaloff>:/images/monitor.png</normaloff>:/images/monitor.png</iconset>
   </property>
   <property name="text">
    <string>Calibration</string>
   </property>
  </action>
  <action name="actionClearPort">
   <property name="icon">
    <iconset resource="../resources/rc.qrc">
     <normaloff>:/images/clear_press.png</normaloff>:/images/clear_press.png</iconset>
   </property>
   <property name="text">
    <string>ClearPort</string>
   </property>
   <property name="toolTip">
    <string>ClearPort</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/rc.qrc"/>
 </resources>
 <connections/>
</ui>
